modules/registry-accessing-metrics.adoc

```diff
@@ -55,8 +61,9 @@ registry configuration.
 
 . Create a cluster role if you do not already have one to access the metrics:
 +
+[source,terminal]
 ----
-$ cat <<EOF |
+$ cat <<EOF | $ oc create -f -
```

Comment:
But don't include the `$` here. Just

`cat <<EOF | oc create -f -`

Reply:
Oh, right. Copy/paste error. I've removed the second `$` but left the first `$` in:
`$ cat <<EOF | oc create -f -`